doc/modules/clustering.rst

```diff
@@ -1249,7 +1261,8 @@ Their harmonic mean called **V-measure** is computed by
   0.51...
 
 The V-measure is actually equivalent to the mutual information (NMI)
-discussed above normalized by the sum of the label entropies [B2011]_.
+discussed above normalized by the arithmetic mean of the label 
```

Comment:
maybe say "when normalized" or "with the aggregation function being the arithmetic mean"?